ts/packages/cli/src/commands/find.ts

```diff
@@ -0,0 +1,175 @@
+import { Args, Flags } from '@oclif/core';
+import path from 'path';
+import { ConfigSchema } from '@configu/node';
+import { createInterface } from 'readline';
+import { createReadStream } from 'fs';
+import { once } from 'events';
+import _ from 'lodash';
+import { TMPL } from '@configu/ts';
+import FastGlob = require('fast-glob');
+import { BaseCommand } from '../base';
+
+export default class Find extends BaseCommand<typeof Find> {
+  private ignoredFiles: Set<string> = new Set(['build', 'dist', 'node_modules']);
+
+  static description = `Finding all config parameters and their usage in the source code based on CFGU files`;
+
+  static examples = [
+    {
+      description: `Find all config parameters and their usage in current directory`,
+      command: `<%= config.bin %> <%= command.id %>`,
+    },
+    {
+      description: `Find all config parameters and their usage in specific directory from provided CFGU file`,
+      command: `<%= config.bin %> <%= command.id %> <PATH_TO_SCAN> --cfgu <PATH_TO_CFGU>`,
+    },
+    {
+      description: `Find all unused config parameters in specific directory`,
+      command: `<%= config.bin %> <%= command.id %> <PATH_TO_SCAN> --unused`,
+    },
+  ];
+
+  static args = {
+    path: Args.directory({
+      description: 'Path to source code',
+      required: true,
+      default: '.',
+    }),
+  };
+
+  static flags = {
+    cfgu: Flags.file({
+      description: 'Path to a specific .cfgu file (default: all CFGU files in the source code)',
+      required: false,
+    }),
+    unused: Flags.boolean({
+      description: 'Only show unused parameters',
+      required: false,
+      default: false,
+    }),
+    'show-templates': Flags.boolean({
+      description: 'Ignore parameters that are parts of templates and treat them as used parameters',
+      required: false,
+      default: false,
+    }),
+    ignore: Flags.string({
```

Comment:
maybe instead of Comma separated just use the multiple option and the the user can do --ignore "" --ignore "" etc.. also I wonder if we need another flag for .ignore files like .gitignore .npmignore etc..